recipes/quast/meta.yaml

```diff
@@ -16,17 +16,23 @@ requirements:
   build:
   - gcc
   - boost {{CONDA_BOOST}}*
+  - perl
```

Comment:
for some historical reasons (and yes we want to change this) we need to depend on perl-threaded